sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -385,8 +385,12 @@ case class LoadDataCommand(
         val hadoopConf = sparkSession.sessionState.newHadoopConf()
         val srcPath = new Path(hdfsUri)
         val fs = srcPath.getFileSystem(hadoopConf)
-        if (!fs.exists(srcPath)) {
-          throw new AnalysisException(s"LOAD DATA input path does not exist: $path")
+        // A validaton logic is been added for non local files, Error will be thrown
+        // If hdfs path doest not exist or if no files matches the wild card defined
+        // in load path
+        if (null == fs.globStatus(srcPath) || fs.globStatus(srcPath).isEmpty) {
+          throw new AnalysisException(s"LOAD DATA input path does not exist " +
+            s"or no files are matching the wildcard string: $path")
```

Comment:
I think the previous message ("LOAD DATA input path does not exist: $path") is fine, it covers the case no path matches the wildcard, like the above case for local path.

Reply:
Fine i think you are right, even the above message will able to convey the issue